assets/queries/cloudFormation/aws/amazon_dms_replication_instance_is_publicly_accessible/query.rego

```diff
@@ -0,0 +1,41 @@
+package Cx
+
+import data.generic.common as common_lib
+import data.generic.cloudformation as cf_lib
+
+CxPolicy[result] {
+	document := input.document
+	resource = document[i].Resources[name]
+	resource.Type == "AWS::DMS::ReplicationInstance"
+	properties := resource.Properties
+	properties.PubliclyAccessible == true
+
+	result := {
+		"documentId": input.document[i].id,
+		"resourceType": resource.Type,
+		"resourceName": cf_lib.get_resource_name(resource, name),
+		"searchKey": sprintf("Resources.%s.Properties.PubliclyAccessible", [name]),
+		"issueType": "MissingAttribute",
```

Comment:
This should be incorrectValeu instead of MissingAttribute as the attribute is present but has the wrong value